### PR TITLE
fix(installer) Mac OSX artefacts do not use zip anymore

### PR DESCRIPTION
**Pull request by johnsudaar**
Fix [SRE-688]

- [ ] Add a changelog entry in the section "To Be Released" of CHANGELOG.md

[SRE-688]: https://scalingo.atlassian.net/browse/SRE-688?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: dists/install.sh [hunk 1]
```diff
@@ -46,10 +46,9 @@ main() {
   uname -a | grep -qi 'Darwin' ; is_darwin=$?
 
   os=$(uname -s | tr '[A-Z]' '[a-z]')
-  ext=zip
+  ext='tar.gz'
```
Comment (EtienneM) [inline dists/install.sh:49]: The switch case after this line could be simplified in a single if, don't you think?

File: dists/install.sh [hunk 2]
```diff
@@ -122,14 +121,7 @@ main() {
...
   status "Extracting...   "
-  case $ext in
-    zip)
-      unzip -d "${tmpdir}" "${tmpdir}/${archive_name}"
-      ;;
-    tar.gz)
-      tar -C "${tmpdir}" -x -f "${tmpdir}/${archive_name}"
-      ;;
-  esac
+  tar -C "${tmpdir}" -x -f "${tmpdir}/${archive_name}"
```
Comment (EtienneM) [inline dists/install.sh:124]: Are we sure `tar` is available on all MacOS?

Comment (johnsudaar) [reply dists/install.sh:124]: I asked @ksol and it ran on his mac.

**Review by EtienneM — APPROVED** [review]

File: dists/install.sh (hunk 2 above)
Comment (ksol) [reply dists/install.sh:118]: pretty sure it's been available for quite some time

**Pull request merged**